### ISSUE TITLE
 VS code instead of Visual Studio

### ISSUE THREAD

**Issue by Chibuike-edmund** [dotnet/intro-to-dotnet-web-dev#38]
Hi,

Thanks a lot for the compilation.

Honestly, I will like to find my way round using the vs code for this tutorial, is there a tutorial for such

**Comment by jamesmontemagno** [comment]
Yes! most of this can be done in VS Code, we will update the instructions. This was originally created specific for VS that is why it is like this today.
